.build/check-broken-links.sh

```diff
@@ -77,8 +88,10 @@ blc --recursive http://127.0.0.1:3000
     --exclude 'https://www.instagram.com/fermyontech/'                                                                                                                                      \
     --exclude 'https://www.linkedin.com/company/fermyon/'                                                                                                                                   \
     --exclude 'https://support.google.com/webmasters/answer/7552505'                                                                                                                        \
-    --exclude 'https://support.google.com/webmasters/answer/9008080?hl=en'| tee /dev/stderr | grep "├─BROKEN─" > broken_links || true
-   
+    --exclude 'https://support.google.com/webmasters/answer/9008080?hl=en' | tee "${report}" || \
```

Comment:
There's probably other ways to fix this, but with the previous logic, false positives were possible.  For instance, if the `blc` process itself exited non-zero, we'd not catch it (per the `|| true`), leading to scenarios like this:
```
$ npm run test
...
Getting links from: http://127.0.0.1:3000/foo
Error: HTML could not be retrieved
All the links are valid!
stopping process for 32479
cleanup_with_child 32479
stopping child 32544 (parent: 32479)
cleanup_with_child 32544
stopping child 32633 (parent: 32544)
cleanup_with_child 32633
stopping child 32698 (parent: 32633)
cleanup_with_child 32698
stopping child 32773 (parent: 32698)
cleanup_with_child 32773
stopping child 32774 (parent: 32773)
cleanup_with_child 32774
stopping self 32774
stopping self 32773
stopping self 32698
stopping self 32633
stopping self 32544

$ echo $?
0

Reply:
good catch!

Reply:
Oh, let me spend a little more time on this.  Now we don't get the nice broken link report when this is the cause for the blc failure. 😅

Reply:
Ok, sorry for the delay.  Now ready for a re-review @karthik2804!